dask/dataframe/backends.py

```diff
@@ -209,10 +209,10 @@ def _(x, index=None):
 
 
 @pyarrow_schema_dispatch.register((pd.DataFrame,))
-def get_pyarrow_schema_pandas(obj):
+def get_pyarrow_schema_pandas(obj, **kwargs):
```

Comment:
I would have strongly preferred that this was an explicit `preserve_index` kwarg. It is typically dangerous to use `**kwargs` in the pandas definition of a dispatch function, because the pandas version is defining the specific options that other backends should handle.

Reply:
>Makes sense, I was following the lead of to_pyarrow_table_dispatch here.

Ah right. Makes sense. I think we were unsure what options the p2p would need at the time, and were planning to revise after things settled.

I suppose we can take the same approach here if things are still in flux. However, I don't want it to be **too** easy to change how p2p uses these dispatch functions since the cudf options are definitely more limited than those in pandas :) (though I will always be understanding of the motivation to address memory issues)